### PR TITLE
Initial 2019-09-01-preview for LVA

### PR DESCRIPTION
**Pull request by BrianBlum**
### Latest improvements:
<i>MSFT employees can try out our new experience at <b>[OpenAPI Hub](https://aka.ms/openapiportal) </b> - one location for using our validation tools and finding your workflow. 
</i><br>
### Contribution checklist:
- [x] I have reviewed the [documentation](https://github.com/Azure/azure-rest-api-specs#basics) for the workflow.
- [x] [Validation tools](https://github.com/Azure/azure-rest-api-specs/blob/master/documentation/swagger-checklist.md#validation-tools-for-swagger-checklist) were run on swagger spec(s) and have all been fixed in this PR.
- [ ] The [OpenAPI Hub](https://aka.ms/openapiportal) was used for checking validation status and next steps.
### ARM API Review Checklist
- [ ] Service team MUST add the "WaitForARMFeedback" label if the management plane API changes fall into one of the below categories. 
- adding/removing APIs.
- adding/removing properties.
- adding/removing API-version. 
- adding a new service in Azure.

Failure to comply may result in delays for manifest application. Note this does not apply to data plane APIs.
- [ ] If you are blocked on ARM review and want to get the PR merged urgently, please get the ARM oncall for reviews (RP Manifest Approvers team under Azure Resource Manager service) from IcM and reach out to them. 
Please follow the link to find more details on [API review process](https://armwiki.azurewebsites.net/rp_onboarding/ResourceProviderOnboardingAPIRevieworkflow.html).

### CODE REVIEW

**Comment by BrianBlum** [comment]
WaitForARMFeedback

**Review by majastrz — APPROVED** [review]
Signed off from ARM side.

File: specification/mediaservices/resource-manager/Microsoft.Media/preview/2019-09-01-preview/MediaGraphs.json [hunk 1]
```diff
@@ -0,0 +1,757 @@
...
+        "x-ms-examples": {
+          "Get a Media Graph by name": {
+            "$ref": "examples/media-graph-get-by-name.json"
+          }
+        }
+      },
+      "put": {
+        "summary": "Create or update a Media Graph",
+        "description": "Create or update a Media Graph in the Media Services account",
+        "operationId": "MediaGraphs_CreateOrUpdate",
```
Comment (anuchandy) [inline specification/mediaservices/resource-manager/Microsoft.Media/preview/2019-09-01-preview/MediaGraphs.json]: Is `creareOrUpdate` a long running operation? same question for `delete`.

File: specification/mediaservices/resource-manager/Microsoft.Media/preview/2019-09-01-preview/MediaGraphs.json [hunk 2]
```diff
@@ -0,0 +1,757 @@
...
+        ],
+        "x-ms-long-running-operation": true,
+        "x-ms-examples": {
+          "Stop Media Graph": {
+            "$ref": "examples/media-graph-stop.json"
+          }
+        }
+      }
+    },
+    "/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.Media/mediaServices/{accountName}/mediaGraphs/{mediaGraphName}/operationsStatus/{operationId}": {
```
Comment (anuchandy) [inline specification/mediaservices/resource-manager/Microsoft.Media/preview/2019-09-01-preview/MediaGraphs.json]: Looks like this is to retrieve result of long running operation, if so usually we don't include this operation instead annotate actual LRO operation with `x-ms-long-running-operation`.

File: specification/mediaservices/resource-manager/Microsoft.Media/preview/2019-09-01-preview/MediaGraphs.json (hunk 1 above)
Comment (PrasannaKudli) [reply specification/mediaservices/resource-manager/Microsoft.Media/preview/2019-09-01-preview/MediaGraphs.json]: Both PUT (CreateOrUpdate) and DELETE are not long running. They complete synchronously. 
In our API, START and STOP actions are Long running operations

File: specification/mediaservices/resource-manager/Microsoft.Media/preview/2019-09-01-preview/MediaGraphs.json (hunk 2 above)
Comment (PrasannaKudli) [reply specification/mediaservices/resource-manager/Microsoft.Media/preview/2019-09-01-preview/MediaGraphs.json]: Both long running operations use "x-ms-long-running-operation": true.  See ling 560 and 608. 
As part of long running operation, we return operationId in response header. User can use this API to get operation status.

**Review by anuchandy — APPROVED** [review]

**Pull request merged**